tests/layer_tests/jax_tests/test_logistic.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import jax
+import numpy as np
+import pytest
+from jax import numpy as jnp
+
+from jax_layer_test_class import JaxLayerTest
+
+rng = np.random.default_rng(5402)
+
+
+class TestLogistic(JaxLayerTest):
+    def _prepare_input(self):
+        
+        input = jnp.array(np.random.uniform(-1000, 1000, self.input_shape).astype(self.input_type))
+        return [input]
+
+    def create_model(self, input_shape, input_type):
+        self.input_shape = input_shape
+        self.input_type = input_type
+
+        def jax_logistic(input):
+            return jax.lax.logistic(input)
+
+        return jax_logistic, None, 'logistic'
+
+    @pytest.mark.parametrize("input_shape", [1, 2, 3, 4, 5, 6, 7, 8, 9, 10])
```

Comment:
pls check other tests. what shapes are used.

Reply:
I have referenced exp, But I have a question why we need to pass shapes like that.

Reply:
In case of `[1, 2, 3, 4, ...]` I don't understand the logic of testing.
In case `[[], [1], [2, 3], ...]` you are testing scalar case, 1D, 2D, etc that are more practical